treeherder/perf/models.py

```diff
@@ -308,22 +309,23 @@ def __init__(self, *args, **kwargs):
 
         # allows updating timestamps only on new values
         self.__prev_bug_number = self.bug_number
+        self.triage_due_date = self.get_time_to_triage()
 
     def save(self, *args, **kwargs):
         if self.bug_number is not None and self.bug_number != self.__prev_bug_number:
-            self.bug_updated = datetime.now()
+            self.bug_updated = datetime.datetime.now()
         super(PerformanceAlertSummary, self).save(*args, **kwargs)
         self.__prev_bug_number = self.bug_number
 
     def update_status(self, using=None):
         self.status = self.autodetermine_status()
+        self.triage_due_date = self.get_time_to_triage()
```

Comment:
@alexandru-io Why is is necessary to update here the `triage_due_date`? As I understand it, the `triage_due_date` field should be calculated only once since it is based on the `created` field, which is not updated after the alert summary is created.

Reply:
when merging to production, the old alerts will have triage date null and the counter will show that this is due since 1900 and will break our KPI calculation for the month. using it here will make sure that when the alert is updated, the due date will be updated correctly. if the function is called repeatedly, it will return the already present due date.